### PR TITLE
FindBug error fixed and some checkstyle issues fixed

### PR DESCRIPTION
**Pull request by Devas**
FindBug error fixed and some checkstyle issues fixed.
All tested and works.

### CODE REVIEW

File: src/main/java/com/fourmusketeers/client/Main.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.fourmusketeers.client;
 
-import com.fourmusketeers.client.View.WelcomeWindow;
+import com.fourmusketeers.client.view.WelcomeWindow;
```
Comment (piotr921) [inline src/main/java/com/fourmusketeers/client/Main.java]: 👍

File: src/main/java/com/fourmusketeers/client/Main.java [hunk 2]
```diff
@@ -10,7 +10,7 @@
  */
 public class Main extends Application {
 
-    private final String WORLD_OF_WARSHIPS = "World of Warships";
+    private static final String WORLD_OF_WARSHIPS = "World of Warships";
```
Comment (piotr921) [inline src/main/java/com/fourmusketeers/client/Main.java]: 👍

File: src/main/java/com/fourmusketeers/client/logic/ShipsGenerator.java [hunk 3]
```diff
@@ -28,7 +28,14 @@
 
     @Override
     public void update(Observable singleTileButton, Object arg) {
-        Integer tileIndex = ((TileButton) singleTileButton).getIndex();
+        Integer tileIndex;
+        if (singleTileButton instanceof TileButton) {
+            tileIndex = ((TileButton) singleTileButton).getIndex();
+        }
+        else {
```
Comment (arturAR) [inline src/main/java/com/fourmusketeers/client/logic/ShipsGenerator.java]: Should be:
} else {

File: src/main/java/com/fourmusketeers/client/logic/ShipValidator.java [hunk 4]
```diff
@@ -50,7 +53,8 @@ void blockShipPosition(Ship ship) {
     }
 
     boolean isFieldAvailable(int tileIndex) {
-        return availableTiles.contains(tileIndex) || availableTiles.isEmpty() && !blockedTiles.get(tileIndex).isBlocked();
+        return availableTiles.contains(tileIndex) ||
+                availableTiles.isEmpty() && !blockedTiles.get(tileIndex).isBlocked();
```
Comment (piotr921) [inline src/main/java/com/fourmusketeers/client/logic/ShipValidator.java]: Demeter law, i would put this in separate methods.

**Pull request merged**